services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/config/PpacConfiguration.java

```diff
@@ -169,6 +169,8 @@ public void setOtp(Otp otp) {
     private String ppacIosJwtSigningKey;
     private Integer minDeviceTokenLength;
     private Integer maxDeviceTokenLength;
+    private Integer apiTokenRateLimitMinutes;
```

Comment:
The validity of the ApiToken it should be in seconds as per new specs.

Reply:
Yes, let's go with seconds please. I'll update the "official" spec.